### ISSUE TITLE
Problem with old downtimes

### ISSUE THREAD

**Issue by duylong** [statusengine/interface#42]
Hi,

I have a problem with Upcoming downtimes.  My old downtimes are still present with the status "Downtime currently running" and "Cancel" button.

![upcoming-downtimes](https://user-images.githubusercontent.com/3019703/81559213-4c6c6500-938f-11ea-957f-7997fee09f35.png)

Do you know why?

**Comment by duylong** [comment]
It seems that when worker is down, the worker does not detect the end downtime.
The worker don't have a auto restart when everything goes wrong??
```
statusengine-worker[13]: Elasticsearch error!
statusengine-worker[13]: Elasticsearch error!
statusengine-worker[13]: No alive nodes found in your cluster
statusengine-worker[13]: No alive nodes found in your cluster
...
statusengine-worker[13]: Elasticsearch error!
statusengine-worker[13]: Elasticsearch error!
statusengine-worker[13]: No alive nodes found in your cluster
statusengine-worker[13]: No alive nodes found in your cluster
````
No problem after a restarted service. 

In any case the interface should still not display expired downtimes.

**Comment by nook24** [comment]
This sounds interesting. Basically this is a worker related issue. How ever, the worker forks a _[MiscChild](https://github.com/statusengine/worker/blob/fb69e99f240b383db5c0bbaaba8599d86c5aba90/bin/StatusengineWorker.php#L83-L86)_ which will handle Notifications, Downtimes and Acknowledgements and a separate _[PerfdataChild](https://github.com/statusengine/worker/blob/fb69e99f240b383db5c0bbaaba8599d86c5aba90/bin/StatusengineWorker.php#L88-L92)_ which process performance data.

Normally whatever happens to the perfdata child should not cause any side effects to other processes / workers.


> In any case the interface should still not display expired downtimes.

I will do some investigation into this.

**Comment by duylong** [comment]
Yes I noticed that I had no side effect with the worker errors. My problem is still present, I can't find the source of the problem to reproduce it. Currently I manually clean the MySQL database, it is not very clean.

**Comment by nook24** [comment]
Is this still a thing?

**Comment by duylong** [comment]
I recently updated to the latest version, I am looking to see if the problem comes back or not ;-)

**Comment by duylong** [comment]
The command "/opt/statusengine/worker/bin/Console.php cleanup" should not clean up old ACK / DOWNTIME?

I still have DOWNTIME from "23:59 12.13.2020"...

**Comment by nook24** [comment]
I noticed this on the Demo System as well. Not sure what exactly caused this, but on my system its only a handful of affected downtimes or acknowledgements over the last years. In this case, the affected records have to be removed from the MySQL tables manually.

Probably this is fixed by now or with the new Go Worker.
